rules/strict-equality.js

```diff
@@ -0,0 +1,66 @@
+const eqeqeq = require("eslint/lib/rules/eqeqeq");
+const astUtils = require("eslint/lib/rules/utils/ast-utils");
+
+// - Prefer double equals when comparing with undefined
+// - Prefer undefined over null
+// - Prefer triple equals everywhere else
+module.exports = {
+  meta: {
+    type: "suggestion",
+    fixable: "code",
+    schema: [],
+    messages: {
+      unexpected: "Prefer '{{expectedOperator}}' over '{{actualOperator}}'.",
+    },
+  },
+
+  create: function (context) {
+    const tripleEq = eqeqeq.create(context);
```

Comment:
Is it normal for rules to import and fall back to other rules like this? It seems like this would lead to errors being reported from rule A that look like they come from rule B (and you’d have to configure/disable B even though your config file only configured A).

Reply:
I don't think it's normal, I haven't seen it done anywhere else. However, our rule is intended as a replacement for `eqeqeq`, and the [source](https://github.com/eslint/eslint/blob/master/lib/rules/eqeqeq.js) for that rule has a bunch of logic in there around correctly handling null and when it is safe to fix. So, we can either duplicate that logic, or fall through to the built in rule. Open to feedback on which approach is beter.